modules/compliance-new-tailored-profiles.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-tailor.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-new-tailored-profiles_{context}"]
+= Creating a new tailored profile
+
+You can write a tailored profile from scratch using the TailoredProfile object. Set an appropriate `title` and `description` and leave the `extends` field empty. Indicate to the Compliance Operator what type of scan will this custom profile generate:
+
+* Node scan: Scans the Operating System.
+* Platform scan: Scans the OpenShift configuration.
+
+Set the following annotation on the TailoredProfile object:
+
+.Example `new-profile.yaml`
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: TailoredProfile
+metadata:
+  name: new-profile
+spec:
+  description: My custom profile <1>
+  title: Custom profile <2>
+compliance.openshift.io/product-type: Node/Platform <3>
```

Comment:
It is a bad example.
It is not the right format for the product-type. The product-type should be included in the annotations.
```
apiVersion: compliance.openshift.io/v1alpha1
kind: TailoredProfile
metadata:
  name: new-profile
  annotations:
    compliance.openshift.io/product-type: Node
spec:
  extends: 
  description: My custom profile 
  title: Custom profile 
```